unraid-ui/src/register.ts

```
@@ -10,25 +10,68 @@ type RegisterParams = {
   pathToSharedCss?: string;
 };
 
+// Type for our simplified Vue component representation
+type CustomElementComponent = {
+  styles?: string[];
+  render?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)